ga4gh/server/static/main.js

```diff
@@ -801,6 +837,7 @@ function readGroupFetcher(geneRequest, geneDataset) {
                     else {
                         let warningMsg = document.getElementById('warningMsg');
                         warningMsg.style.display = "block";
+                        warningMsg.innerHTML = '<a href="#" class="close" data-dismiss="alert" aria-label="close">×</a>'
```

Comment:
these code seem repetitive, they should be calling a global helper to display the warning message